docker/web.Dockerfile

```diff
@@ -3,18 +3,20 @@ FROM golang:alpine
 # Required because go requires gcc to build
 RUN apk add build-base
 
+RUN apk add --no-cache git
+
 RUN apk add inotify-tools
 
 RUN echo $GOPATH
 
 COPY . /clean_web
 
-RUN go get github.com/rubenv/sql-migrate/...
+RUN go install github.com/rubenv/sql-migrate/...@latest
```

Comment:
@paudelgaurav Let's move installation instructions before modifying folder structure (`COPY` command). Doing this will ensure that the commands before copy are cached and rebuild is faster.

Reply:
Sure, I have moved up the installation instruction before `COPY` command in commit: 4e0142de9e1131f332f4e8e972173c7d53c28b05